web-components/src/components/form/RecipientsForm.tsx

```diff
@@ -38,7 +39,7 @@ export interface FormValues {
   recipients: Recipient[];
 }
 
-const validationSchema = Yup.object().shape({
+export const validationSchemaFields = {
   recipients: Yup.array()
     .of(
       Yup.object().shape({
```

Comment:
regarding the TODO just below for validating the recipient address, you can now use `Bech32Address.validate(...)` function (see `RolesForm.tsx`)